index.html

```diff
@@ -0,0 +1,366 @@
+            <span class="number">4</span>
+            <span class="accordText">Secure Credentials</span>
+            <button class="accordion"></button>
+          </div>
+          <div class="panel">
+            <img src="image/groop4.png" alt="#" class="mainIcon" />
+            <div class="footerAccordionContainer">
+              <div class="footerAccordion">
+                <span
+                  >There’s nothing hackers love more than a dismissed update
+                  reminder. We keep our team’s computers in shape and up to date
+                  so that they aren’t subject to attacks that could otherwise be
+                  avoided.</span
+                >
+                <span
+                  >Not properly updating your systems makes you an easy to reach
+                  target and hackers will often go after the lowest hanging
+                  fruit. When it comes to maintaining a strong cybersecurity
+                  network the best thing a user can do is to prevent problems
+                  before they occur, and the easiest way to do that is by making
+                  sure that all systems are updated and updated regularly.
+                </span>
+              </div>
+              <div class="bottomIcon">
+                <img src="image/Letter_i.png" alt="#" />
+                <span>Based on the 2018 Verizon Data Breach Report</span>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="line"></div>
+
+        <div class="glossaryContaner">
+          <div class="line"></div>
+          <img src="image/skout-glossary.svg" id="headGlossary" alt="#" />
+        </div>
+      </div>
+
+      <section>
+        <div class="sliderContainer">
+          <div class="sliderHead"></div>
+          <div class="sliderButtonContainer">
+            <button class="sliderButton" value="right"></button>
+            <button class="sliderButton rotateArrow" value="left"></button>
+          </div>
+        </div>
+        <div class="sliderTextContainer">
+          <div>
+            <p class="sliderP">Ransomware</p>
+          </div>
+          <div class="sliderText">
+            <div class="line"></div>
+            <span class="sliderSpan"
+              >Malicious software that blocks access to a system until a ransom
+              is paid.</span
+            >
+          </div>
+        </div>
+      </section>
+
+      <div class="faqsContainer">
+        <div class="line"></div>
+        <img src="image/skout-faqs.svg" id="headFAQs" alt="" />
+      </div>
+
+      <div class="faqsAccordionContainer">
+        <div class="">
+          <div class="headAccordionFaqs">
+            <span>Q</span>
+            <span class="question">What are your system requirements?</span>
+            <button class="accordion"></button>
+          </div>
+          <div class="panel">
+            <span class="panelSpan">A</span>
+            <div>
```

Comment:
can be simplify to `p` whithout extra wrapper

Reply:
done